source/connect/connection-options.txt

```diff
@@ -18,4 +18,316 @@ Specify Connection Options
    :keywords: connection string, URI, server, Atlas, settings, configure
 
 Overview
---------
\ No newline at end of file
+--------
+
+This section describes the MongoDB connection and authentication options
+available in the {+driver-short+}. You can configure your connection by
+setting options in either the connection URI or within a
+``MongoClientSettings`` instance.
+
+.. _kotlin-sync-connection-uri-settings:
+
+Set Options in the Connection URI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you pass a connection URI to the ``MongoClient.create()`` method, you can include
+connection options in the string as ``<name>=<value>`` pairs. In the following example,
+the connection URI contains the ``connectTimeoutMS`` option with a value of ``60000``
+and the ``tls`` option with a value of ``true``:
+
+.. code-block:: kotlin
+
+   val uri = "mongodb://<hostname>:<port>/?connectTimeoutMS=60000&tls=true"
+   val mongoClient = MongoClient.create(uri)
+
+.. _kotlin-sync-mongoclientsettings:
+
+Set Options in MongoClientSettings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can set connection options in a ``MongoClientSettings`` instance by
+using methods from the ``MongoClientSettings.Builder`` class, then passing
+the settings object to the ``MongoClient.create()`` method.
+
+Configuring the connection this way makes it easier to
+change settings at runtime and can help you catch errors at compile time.
+
+The following example shows how to set options when creating a
```

Comment:
Recommend adding a section just using the MongoClientSettings.builder eg:

```
        MongoClientSettings
            .builder()
            .applyToClusterSettings { it.hosts(listOf(ServerAddress("127.0.0.1"))) }
            .applyToSocketSettings { it.connectTimeout(60000, TimeUnit.MILLISECONDS) }
            .applyToSslSettings { it.enabled(true) }
            .build()
```

Then give the example below mixing a connection string and settings.